pypgstac/pypgstac/migrations/pgstac.0.3.3-0.3.4.sql

```diff
@@ -104,17 +104,17 @@ BEGIN
 
             prev_area := unionedgeom_area;
 
-            RAISE NOTICE '% % % %', st_area(unionedgeom)/tilearea, counter, scancounter, ftime();
+            RAISE NOTICE '% % % %', unionedgeom_area/tilearea, counter, scancounter, ftime();
             IF fields IS NOT NULL THEN
                 out_records := out_records || filter_jsonb(iter_record.content, includes, excludes);
             ELSE
                 out_records := out_records || iter_record.content;
             END IF;
 
-            IF counter > _limit
+            IF counter >= _limit
                 OR scancounter > _scanlimit
                 OR ftime() > _timelimit
-                OR unionedgeom_area >= tilearea
+                OR (skipcovered AND unionedgeom_area >= tilearea)
```

Comment:
should only check for tile coverage if skipcovered==True

Reply:
Actually this call should not include the skipcovered. This is the statement that returns as soon as we have covered the entire area of the geometry passed in. The skipcovered parameter is there to allow us to not include an individual item where that item would be "buried" by the other items that have already been included. I'll change this back.